cluster/gce/gci/configure-helper.sh

```diff
@@ -995,6 +995,13 @@ users:
 EOF
   fi
 
+  if [[ "${ENABLE_APISERVER_TRACING:-false}" == "true" ]]; then
+    cat <<EOF >/etc/srv/kubernetes/opentelemetry_config.yaml
+apiVersion: apiserver.k8s.io/v1alpha1
```

Comment:
config types use `*.config.k8s.io`, so `apiserver.config.k8s.io/...`

using the normal decoding path will catch this

Reply:
Interesting... I assumed this belonged in the same place as the EgressSelectorConfiguration.  Does that mean I should extract the type out of staging/src/k8s.io/apiserver/pkg/apis/apiserver, and move it somewhere else?

Reply:
Figured it out.  The apiserver.k8s.io group was only used for v1alpha1 and v1beta1 apis, and the v1 group had moved to apiserver.config.k8s.io.  I had to add apiserver.config.k8s.io to the v1alpha1 and v1beta1 groups.

Done.